lib/ui/window.dart

```diff
@@ -328,7 +300,7 @@ class FlutterWindow extends FlutterView {
 /// `WidgetsBinding.instance.platformDispatcher` over a static reference to
 /// [window], or [PlatformDispatcher.instance]. See the documentation for
 /// [PlatformDispatcher.instance] for more details about this recommendation.
-class SingletonFlutterWindow extends FlutterWindow {
+class SingletonFlutterWindow extends FlutterView {
```

Comment:
I'm wondering if inheritance is a good idea here. I don't think a window _is_ a view. I propose that we keep `FlutterWindow` and let it own a `FlutterView view` property. This is probably a less breaking change as well, so that we can discuss how to refine it in phase 2.

Reply:
Unless this is actively breaking anything I think this is fine. SingletonFlutterWindow was a hack to avoid a massive breaking change. It will get deprecated in the next few weeks as we continue to remove the single window assumption from the framework.